### DISCUSSION TITLE
Suggestion:provide a config for functional to config the uploadfile's dir

### DISCUSSION THREAD

**Discussion by JunelanMe** [Codeception/Codeception#6177] [Ideas]
debug the attachFile to see the tmpfile 's dir .
I found that the temp file get from UploadedFile::getInstance return the value ,it give the tmpname of the  file we attachFile just now . the path is "tempname=''/tmp/xxxxxxxxxxxxxxx.pdf"  not the path we define in php.ini .   [as we tmp dir usually need the root user to deal the file ] 

why don.t we provide  a config for funtional to config the uploadfile's dir

**Comment by Naktibalda** [comment]
A problem with that is that "functional" is not a single module - it is at least 12 modules (PhpBrowser + all the frameworks). They all inherit from InnerBrowser, but there is no shared configuration and whoever takes this task will have to modify all modules.
